pkg/model/BUILD.bazel

```diff
@@ -53,11 +53,13 @@ go_library(
         "deployment.go",
         "docs.go",
         "environment.go",
-        "event.go",
         "filestore.go",
         "image_name.go",
+        "imagemetadata.pb.go",
+        "imagemetadata.pb.validate.go",
```

Comment:
Maybe you should perform `make clean`...

Reply:
Nice catch!